brain/converter/crypto.py

```diff
@@ -5,12 +5,12 @@
 
 '''
 
-
+from flask import current_app
 import os
 import hashlib
 import base64
 
-saltlength = 32
+salt_length = int(current_app.config.get('SALT_LENGTH'))
```

Comment:
You shouldn't need to cast this to an `int` if the value stored yaml file is already an integer.